Add introduction.md

KTales opened the pull request.

Commit:
Add introduction.md

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1 @@
+Hello! This is my introduction for the Mastering Github course!
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Commit:
Modified introduction.md

Commit:
Modified introduction.md AGAIN

Commit:
Modify introduction.md. Added list of deadly skills

Commit:
Modify introduction.md - added new skill

codeschool-kiddo committed:
Added new rule about coffee

KTales committed:
Merge pull request #1 from KTales/deadly_skills

Modify introduction.md. Added list of deadly skills

Commit:
Fixed dojo_rules.md - typos

Commit:
Done

branch 'new_rules'

# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.
